sdks/python/apache_beam/options/pipeline_options.py

```diff
@@ -1535,9 +1535,8 @@ def _add_argparse_args(cls, parser):
     parser.add_argument(
         '--spark_version',
         default='3',
-        choices=['3', '2'],
-        help='Spark major version to use. '
-        'Note, Spark 2 support is deprecated')
+        choices=['3'],
```

Comment:
Do you keep it for future Spark versions, like `Spark 4`?

Reply:
No, removing the pipeline option would be a breaking change for the ones using spark 3 and setting this explicitly.